src/Apps/Order/Routes/__tests__/Shipping.jest.tsx

```diff
@@ -91,53 +88,25 @@ jest.mock("relay-runtime", () => ({
   ...jest.requireActual("relay-runtime"),
   commitMutation: (...args) => mockCommitMutation(args),
 }))
+jest.mock("react-relay", () => ({
+  ...jest.requireActual("react-relay"),
+  commitMutation: (...args) => mockCommitMutation(args),
+}))
 
-const testOrder: ShippingTestQuery$rawResponse["order"] = {
+const order: ShippingTestQuery$rawResponse["order"] = {
   ...UntouchedBuyOrder,
   internalID: "1234",
   id: "1234",
 }
 
-const ArtsyShippingDomesticFromUSOrder: ShippingTestQuery$rawResponse["order"] = {
-  ...UntouchedBuyOrderWithArtsyShippingDomesticFromUS,
-  __typename: "CommerceBuyOrder",
-  mode: "BUY",
-  internalID: "1234",
-  id: "1234",
-}
-
-const ArtsyShippingInternationalFromUSTestOrder: ShippingTestQuery$rawResponse["order"] = {
-  ...UntouchedBuyOrderWithArtsyShippingInternationalFromUS,
-  __typename: "CommerceBuyOrder",
-  mode: "BUY",
-  internalID: "1234",
-  id: "1234",
-}
-
-const ArtsyShippingDomesticFromGermanyOrder: ShippingTestQuery$rawResponse["order"] = {
-  ...UntouchedBuyOrderWithArtsyShippingDomesticFromGermany,
-  __typename: "CommerceBuyOrder",
-  mode: "BUY",
-  internalID: "1234",
-  id: "1234",
-}
-
-const ArtsyShippingInternationalFromGermanyOrder: ShippingTestQuery$rawResponse["order"] = {
-  ...UntouchedBuyOrderWithArtsyShippingInternationalFromGermany,
```

Comment:
These customizations seem unnecessary, so I just used the untouched ones to simplify fixture setup.